### PR TITLE
CCSMB-3: Folders

### PR DESCRIPTION
**Pull request by tomodachi94**
Imported from oeed/CraftOS-Standards.

### CODE REVIEW

File: Standards/CCSMB-3.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# CCSMB-3: Directory
+
+*Author: Oliver Cooper (oeed)*
+*Last updated: 2017-01-27*
+*Version: v1.0.0*
```
Comment (Ocawesome101) [inline Standards/CCSMB-3.md:5]: these should be spaced further apart - as-is they appear on the same line

**Review by Ocawesome101 — CHANGES_REQUESTED** [review]

File: Standards/CCSMB-3.md (hunk 1 above)
Comment (tomodachi94) [reply Standards/CCSMB-3.md:5]:
```suggestion
*Author: Oliver Cooper (oeed)*

*Last updated: 2017-01-27*

*Version: v1.0.0*
```

Fixed.

**Review by Ocawesome101 — APPROVED** [review]

File: Standards/CCSMB-3.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| Information |                           |
+| ----------- | ------------------------- |
+| Version     | 1.0.0                     |
+| Type        | Folder/directory          |
+| MIME        | `application/directory`   |
+| Extensions  | *See 'Extensions'*        |
+
+## Extensions
+
+In short: *anything*.
```
Comment (MCJack123) [inline Standards/CCSMB-3.md:20]: Making a standard and then saying "this can be anything at all" isn't really standardization.

**Review by MCJack123 — CHANGES_REQUESTED** [review]
I don't think this "standard" is really necessary to bring over at all. All it does is define a MIME type, which we haven't actually really used anywhere else except for the few legacy standards.

**Comment by tomodachi94** [comment]
> I don't think this "standard" is really necessary to bring over at all. All it does is define a MIME type, which we haven't actually really used anywhere else except for the few legacy standards.

Very good point. I'll close this in 24 hours if there are no more objections.

**Comment by MCJack123** [comment]
Closed due to inactivity.

**Pull request closed without merging**